.nix/coq-overlays/mathcomp-warnings/default.nix

```
@@ -0,0 +1,7 @@
+{ mathcomp }: (mathcomp.override { single = true; }).overrideAttrs {
+  preConfigure = ''
+    echo "-arg -w -arg +default" > mathcomp/Make.tmp
+    cat mathcomp/Make >> mathcomp/Make.tmp
+    mv mathcomp/Make.tmp mathcomp/Make
```

This should also work

```suggestion
    echo -e "\n-arg -w -arg +default" >> mathcomp/Make
```

---

I mean, I think that putting it last ensure it overrides what comes before. Don't we want that?

---

No, that was also my itinial try but we do silence some warnings for a reason.